src/events.h

```diff
@@ -60,6 +59,13 @@ namespace events {
     };
     extern signal<void(StickData &)> stick_event;
 
+    // Audio
+    struct AudioData {
+        std::string filename;
+    };
```

Comment:
Are u planning to have other variables in this struct? If not, then just the string would be fine